src/read_lakeparam.c

```diff
@@ -167,6 +167,13 @@ lake_con_struct read_lakeparam(FILE            *lakeparam,
       sprintf(tmpstr, "Lake area fraction (%f) for cell (%d) specified in the lake parameter file must be a fraction between 0 and 1.", temp.Cl[0], soil_con.gridcel);
       nrerror(tmpstr);
     }
+    if(fabs(1-temp.Cl[0]/veg_con[temp.lake_idx].Cv) > 0.01) {
```

Comment:
Actually hold off on that - since 4.2 has not been reformatted anyway. @jhamman : Should this go on a hit fix branch first or can this go directly into master?